README.md

```diff
@@ -25,10 +25,15 @@ git clone --depth 1 -b "v0.6.0" --single-branch https://github.com/k8s-school/ki
 cd kind-travis-ci
 ./kind/k8s-create.sh -s
 
+# Install Qserv in two lines
+curl -fsSL https://raw.githubusercontent.com/lsst/qserv-operator/tickets\/DM-24372/deploy/qserv.sh | bash -s
+kubectl apply -k https://github.com/lsst/qserv-operator/base
+
+# Or include launch of integration tests
 cd "$WORKDIR"
 git clone  https://github.com/lsst/qserv-operator
 cd qserv-operator
-./deploy.sh
+./deploy/qserv.sh
 ./wait-operator-ready.sh
 kubectl apply -k base
 ./wait-qserv-ready.sh
```

Comment:
does this sequence launch the integration tests now? Or, I'm confused about line 32 "include launch of integration tests"

Reply:
No it does not. Fixed.